classes/class-loader.php

```diff
@@ -110,6 +110,22 @@ private function __construct() {
 
 		// Load block patterns.
 		Block_Pattern::register_all();
+
+		// Load parallax library for Media & Text block.
+		add_action(
+			'wp_enqueue_scripts',
+			static function () {
```

Comment:
What is the reason to implement a `static` function?

Reply:
Tbh I don't know 😬 I've removed it since it seems to work fine without it!

Reply:
[In theory](https://www.php.net/manual/en/functions.anonymous.php#functions.anonymous-functions.static) you avoid any binding issue with the container class (and it might have a tiny tiny perf advantage), in practice for this case both are fine :)

Reply:
Okay, thanks for the explanation Flo 👍 removed the commit!

Reply:
Interesting @lithrel and sorry @mleray for this confusion 🤭